source/core/parameters.cc

```diff
@@ -1067,13 +1083,28 @@ namespace Parameters
         Patterns::Selection("constant|immersed solid tanh"),
         "Model used for the calculation of the tracer diffusivity"
         "Choices are <constant|immersed solid tanh>.");
+
       prm.declare_entry(
         "tracer diffusivity",
         "0",
         Patterns::Double(),
         "Tracer diffusivity for the fluid corresponding to Phase = " +
           Utilities::int_to_string(id, 1));
 
+      prm.declare_entry(
+        "tracer reaction constant model",
+        "constant",
+        Patterns::Selection("constant|immersed solid tanh"),
+        "Model used for the calculation of the tracer reaction constant"
+        "Choices are <constant|immersed solid tanh>.");
+
+      prm.declare_entry(
+        "tracer reaction constant",
+        "0",
+        Patterns::Double(),
+        "Tracer first order reaction constant for the fluid corresponding to Phase = " +
```

Comment:
I am not sure, but is it the reaction constant that is first order, or the reaction itself?